### ISSUE TITLE
[Bug]: Can't restart the testing docker stack

### ISSUE THREAD

**Issue by nboisteault** [3liz/lizmap-web-client#3545]
### What is the bug?

Can't restart the testing docker stack.

### Steps to reproduce the issue

I can start the testing docker stack the first time without any issue but when I `./run-docker stop` than `./run-docker` I have in logs:

```bash
lizmapmaster_test_qgis  | Running as 1000:1000
lizmapmaster_test_qgis  | rm: cannot remove '/tmp/.X99-lock': Operation not permitted
```

### Versions

master

### Check Lizmap plugin

- [ ] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu

### Browsers

Chrome

### Browsers version

All

**Comment by dmarteau** [comment]
Consider updating your qgis-server image.

**Comment by nboisteault** [comment]
I did `docker compose pull`. Nothing changed.

**Comment by dmarteau** [comment]
make a `./run-docker reset` then try to `./run-docker` again

**Comment by rldhont** [comment]
I done this
```bash
./run-docker down
./lizmap-ctl clean
docker volume rm lizmapmaster_pg_data
./run-docker build
./run-docker
./lizmap-ctl install
./qgis-projects/tests/load_sql.sh
./run-docker stop map
./run-docker
```
And the container `map` does not restart
```bash
./run-docker logs map
lizmapmaster_test_qgis  | Running as 1000:1000
lizmapmaster_test_qgis  | rm: cannot remove '/tmp/.X99-lock': Operation not permitted
```

**Comment by nboisteault** [comment]
> make a `./run-docker reset` then try to `./run-docker` again

@dmarteau It works the first time but the error comes back after `./run-docker stop` and `./run-docker`.

**Comment by dmarteau** [comment]
This is - partially - fixed in 3liz/py-qgis-server@689d330c044a7ac3da4d648152adaac24b22e2ce.

To make the story short : You should be ok when running the container as root and setting the variable `QGSRV_USER` - which the case here, afaik.   But if your try to restart a container started with `--user` option, you will have to add the `--tmpfs /tmp` option (as stated int the error message if this is not the case).

**Comment by nboisteault** [comment]
@dmarteau should it be ok now?
